nxc/protocols/winrm.py

```diff
@@ -348,11 +348,11 @@ def ps_execute(self):
 
     def sam(self):
         try:
-            self.execute("reg save HKLM\SAM C:\\windows\\temp\\SAM && reg save HKLM\SYSTEM C:\\windows\\temp\\SYSTEM", False)
+            self.execute("cmd /c 'reg save HKLM\SAM C:\\windows\\temp\\SAM && reg save HKLM\SYSTEM C:\\windows\\temp\\SYSTEM'", False)
```

Comment:
bad idea, you will have this error everytime

![image](https://github.com/Pennyw0rth/NetExec/assets/5891788/d12813d9-ac49-4f98-9420-f2c58cfd92bd)


remove the cmd.exe /c

Reply:
sure

Reply:
Now it should be ok, I just add `--dump-method` to make user dump over cmd or powershell

Reply:
In powershell, it need `cmd /c`, because `&&` will break powershell